### PR TITLE
MAPI-1436 - Tag name appears in Euro minute notifications

### PR DESCRIPTION
**Pull request by davidfurey**
Don't include topic name in euro 2016 minute alerts

### CODE REVIEW

File: src/test/scala/com/gu/mobile/notifications/content/lib/ContentAlertPayloadBuilderSpec.scala [hunk 1]
```diff
@@ -141,6 +141,14 @@ class ContentAlertPayloadBuilderSpec extends WordSpec {
       assert(ContentAlertPayloadBuilder(minuteItem) === Some(expectedMinutePayload))
     }
 
+    "do not prefix title for euro 2016 minute" in {
+      val euroTag = Tag("football/series/the-uefa-euro-minute-2016", "series", None, None, "Steve", "", "")
+      val euroTopic = Topic(TagSeries, "football/series/the-uefa-euro-minute-2016")
+      val minuteItem = item.copy(tags = List(euroTag))
+      val expectedMinutePayload = expectedPayloadForItem.copy(title = "headline", topic = Set(euroTopic))
+      assert(ContentAlertPayloadBuilder(minuteItem) === Some(expectedMinutePayload))
```
Comment (agulowaty) [inline src/test/scala/com/gu/mobile/notifications/content/lib/ContentAlertPayloadBuilderSpec.scala]: `ContentAlertPayloadBuilder(minuteItem) must beSome()` ?

Comment (davidfurey) [inline src/test/scala/com/gu/mobile/notifications/content/lib/ContentAlertPayloadBuilderSpec.scala]: scalatest :-(

File: src/main/scala/com/gu/mobile/notifications/content/lib/ContentAlertPayloadBuilder.scala [hunk 2]
```diff
@@ -57,14 +58,18 @@ trait ContentAlertPayloadBuilder {
       )
   }
 
-  private def contentTitle(content: Content, followableTag: Option[Tag], topics: Set[Topic]): String =
+  private def contentTitle(content: Content, followableTag: Option[Tag], topics: Set[Topic]): String = {
+    def title = content.fields.getOrElse(Map()).getOrElse("headline", content.webTitle)
+    def reason = followableTag.map(_.webTitle).getOrElse("Following")
```
Comment (agulowaty) [inline src/main/scala/com/gu/mobile/notifications/content/lib/ContentAlertPayloadBuilder.scala]: I'd leave `reason` where it was previously

**Comment by agulowaty** [comment]
👍

**Pull request merged**